driver/normatool/validator.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2024 Fantom Foundation
+// This file is part of Norma System Testing Infrastructure for Sonic.
+//
+// Norma is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Norma is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with Norma. If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"path"
+
+	"github.com/Fantom-foundation/go-opera/evmcore"
+	"github.com/Fantom-foundation/go-opera/inter/validatorpk"
+	"github.com/Fantom-foundation/go-opera/valkeystore"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/urfave/cli/v2"
+)
+
+// fakeKey returns a hex privatekey from a list of prepared fake privatekey
+func fakeKey(n uint32) (k *ecdsa.PrivateKey, retErr error) {
```

Comment:
It can, but the function itself panics.
https://github.com/Fantom-foundation/Sonic/blob/ce6dcab136ddf8d2c874b08b6e563820b91abe74/evmcore/apply_fake_genesis.go#L137

This is an attempt to work around that. Do you think it's unnecessary?

Reply:
Discussed. Keep this until issue arises.